src/utils/env.ts

```diff
@@ -23,6 +23,14 @@ export function isAppUniswapStagingOrg({ hostname }: { hostname: string }): bool
   return hostname === 'app.uniswap-staging.org'
 }
 
+export function isBrowserRouterEnabled(): boolean {
+  return (
+    isAppUniswapOrg(window.location) ||
+    isAppUniswapStagingOrg(window.location) ||
+    window.location.hostname === 'localhost'
```

Comment:
I think we have an `isDev` node env hook to replace checking for localhost

Reply:
Theoretically yes, but what if we push it to some external location in DEV mode (for whatever reason) and it breaks? Kind of an edge case, but just thinking out loud.

To be honest, I was thinking to invert this. Rather than checking for Uniswap/hostname, localhost and Vercel (clearly a lot of environments), check if it's IPFS instead. That would require just one check.

Reply:
Okay, I think I will replace with `isDev` instead, b/c all IPFS builds won't have that flag anyway. Will also add Vercel to the list.